node/forcedroid.js

```diff
@@ -300,7 +291,6 @@ function createNativeApp(config, showNextSteps) {
     // copy <Android Package>/libs/SalesforceSDK -> <App Folder>/forcedroid/libs/SalesforceSDK
     var salesforceSDKRelativePath = path.join('libs', 'SalesforceSDK');
     copyFromSDK(packageSdkRootDir, config.targetdir, salesforceSDKRelativePath);
```

Comment:
Turns out you can't specify a repo path for Gradle dependency - it has to be on MavenCentral, so until we publish, we need a physical copy to link to. Also, the generated project needs a ```settings.gradle``` at the root level that specifies the projects, which I need to generate for a template app that we generate, and then I need to tweak the ```build.gradle``` files to reflect the correct relative path. It's easier than ```ant```, because there's only 1 line in 1 file to tweak (for native). We can get to a point where none of this necessary once we get on ```MavenCentral```.